src/it/JENKINS-58771-packaged-plugins/invoker.properties

```
@@ -0,0 +1,3 @@
+# install, not verify, because we want to check the artifact as we would be about to deploy it
+# release.skipTests normally set in jenkins-release profile since release:perform would do the tests
+invoker.goals=-Pjenkins-release -Drelease.skipTests=false clean install
```

```suggestion
invoker.goals=clean install
```

should suffice here. I think you are copying from the one IT that tried to check additional stuff.